modules/dist/pom.xml

```diff
@@ -93,44 +46,50 @@
     </build>
 
     <profiles>
-
-    <!-- 
-    Name:  download
-    Descr: Download WildFly  
-    -->
     <profile>
-      <id>download</id>
+      <id>provisioning</id>
       <activation>
         <property>
           <name>!server.home</name>
         </property>
       </activation>
       <build>
-        <plugins>
-          <plugin>
-            <artifactId>maven-dependency-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>unpack</id>
-                <phase>validate</phase>
-                <goals>
-                  <goal>unpack</goal>
-                </goals>
-                <configuration>
-                  <artifactItems>
-                    <artifactItem>
-                      <groupId>org.wildfly</groupId>
-                      <artifactId>wildfly-dist</artifactId>
-                      <version>${jboss.version}</version>
-                      <type>zip</type>
-                      <overWrite>false</overWrite>
-                      <outputDirectory>${project.build.directory}</outputDirectory>
-                    </artifactItem>
-                  </artifactItems>
-                </configuration>
-              </execution>
-            </executions>
-          </plugin>
+          <plugins>
```

Comment:
this has different indentation than previously

Reply:
Fixed.